index.html

```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head lang="en">
-    <meta charset="UTF-8">
-    <meta http-equiv="refresh" content="0; url=doc/" />
```

Comment:
Deleted this file because all it does is redirect to `doc` which is not helpful.